doc/manual/src/command-ref/files/user-profiles.md

```diff
@@ -0,0 +1,157 @@
+# User profiles
```

Comment:
Please incorporate my fixes to `nix profile --help` I think you removed by accident. The user of XDG dirs for the profiles directory is unconditional.

Reply:
Are there any other changes apart from the ones you've mentioned?

Reply:
I mean #8078, the changes from that PR are gone.

Reply:
You might also double check #8114 but I think those are OK.

Reply:
Thanks. I have incorporated them back.